### ISSUE TITLE
Unclear onboarding message for "storage template engine"

### ISSUE THREAD

**Issue by Cretezy** [immich-app/immich#7129]
### The bug

This message is unclear in the onboarding. What happens if the storage template engine is off? Does it use the default settings? This should be explained here

![image](https://github.com/immich-app/immich/assets/2672503/0abd2e88-dd5d-4c69-9503-5f8f8d4ef317)

### The OS that Immich Server is running on

Docker, Unraid

### Version of Immich Server

v1.19.4

### Version of Immich Mobile App

v1.19.4

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
Go through onboarding setup.
```

**Comment by mertalev** [comment]
You're right that we should explain what happens when it's disabled. It means that when assets are uploaded to the `upload` folder, they stay there instead of being moved based on the storage template. 

The reason you might prefer this is if the move would be slow. In most cases, the move is done instantly by "renaming" the file to be in a different path. But depending on the file system, whether those two folders are in the same drive, whether they're network storage, etc., the whole file may need to be copied. This can cause a lot of disk wear, hence why it's opt-in.
